test/models/user_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class UserTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
```

Comment:
@vlasiak, It looks you missed to add some tests here ;) I expect that a user, at least, should have not blank and unique email (didn't find such validations at model level).